src/system.rs

```diff
@@ -810,18 +810,20 @@ macro_rules! system {
             /// * `E`: `typenum::Integer` power.
             #[inline(always)]
             pub fn powi<E>(
-                self, e: E
+                self, _e: E
             ) -> Quantity<$quantities<$($crate::typenum::Prod<D::$symbol, E>),+>, U, V>
             where
                 $(D::$symbol: $crate::lib::ops::Mul<E>,)+
                 D::Kind: $crate::marker::Mul,
                 E: $crate::typenum::Integer,
-                V: $crate::typenum::Pow<E, Output = V> + $crate::Conversion<V>,
+                V: $crate::Conversion<V>,
             {
+                use $crate::ConversionFactor;
+
                 Quantity {
                     dimension: $crate::lib::marker::PhantomData,
                     units: $crate::lib::marker::PhantomData,
-                    value: $crate::typenum::Pow::powi(self.value, e),
+                    value: self.value.into_conversion().powi(E::to_i32()).value(),
```

Comment:
This is probably just a lack of understanding, but looking at e.g. the implementation of `recip`, why is the trait bound
```
V: $crate::num::Float
```
and the implementation
```
value: self.value.powi(E::to_i32()),
```
not applicable here?

Reply:
I thought I test this when I did the original fix, but I must have been pretty distracted because I checked it again and it works. I just pushed the fix and am waiting on the build to finish. Thanks for the review!